specification/batch/Azure.Batch/tspconfig.yaml

```diff
@@ -0,0 +1,28 @@
+parameters:
+  "service-dir":
+    default: "sdk/batch"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "./data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/{service-name}/preview/{version}/BatchService.json"
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Batch"
+    clear-output-folder: true
+    model-namespace: false
+    head-as-boolean: true
+    namespace: Azure.Compute.Batch
+  "@azure-tools/typespec-java":
+    namespace: "com.azure.compute.batch"
+    package-dir: "Batch"
```

Comment:
If namespace is com.azure.compute.batch, package-dir should be `azure-compute-batch`

Reply:
thanks, that fixed the issue